packages/paste-website/src/pages/patterns/empty-states/index.mdx

```diff
@@ -0,0 +1,444 @@
+        <Anchor href="../components/paragraph">Paragraph</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/button">Button (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/anchor">Anchor (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../illustrations">Illustrations (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import all ingredients for the empty state patterns
+import {​Card} from '@twilio-paste/core/card';
+import {Heading} from '@twilio-paste/core/heading';
+import {Paragraph} from '@twilio-paste/core/paragraph';
+import {Button} from '@twilio-paste/core/button';
+import {​Anchor} from '@twilio-paste/core/anchor';
+```
+
+## Usage
+
+### General
+
+This pattern is used to explain to a customer what they should do next if content is either unavailable or can't be displayed.
+
+Empty states should:
+
+- **Be clear:** If there's a path forward, explain it as concisely as possible. Include an explanation for why a customer is seeing the empty state if it helps them better understand what to do next.
+- **Give direction and educate the customer:** Let customers know what steps they can take to move forward in a flow. Guiding them to "Create an alpha sender" is more helpful than saying "No alpha senders".
+- **Be encouraging and inspire confidence:** Use a [friendly and approachable tone](/content/voice-and-tone#encouraging) to ensure a positive experience. Never make customers feel unsuccessful or guilty because they're seeing an empty state. Even when they arrive at a page "in error" (for example, on a page they don't have permissions to), encouraging messaging and an explanation of possible solutions can give customers confidence that they're heading in the right direction.
+
+In most cases, treat an empty state as an informational state of the entire page, and not a state that simply replaces a part of the page that should otherwise be populated. When you have text explaining what a page is for, try working it into the empty state card.
+
+When a page is in an empty state, its call-to-action (CTA) should appear in the empty state card, not near the page heading like in the [Create pattern](/patterns/create/). Exceptions can be made in cases like some "No results found" scenarios, when a CTA is already on the page on load. The visually focus of hiding the CTA that's outside of the card could draw too much of the customer's attention away from their primary goal.
+
+<Box marginBottom="space70">
```

Comment:
I'm really liking these code sandboxes included w/in the page. I think it makes it easy to see how the pattern is created and will be super useful!!